tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py

```diff
@@ -108,8 +108,8 @@ def test_settings_app(self):
         settings.return_to_prev_menu(cs_page.cs_screen_element, cs_page.call_forwarding_screen_element)
 
         cs_page.tap_call_barring()
-        self.take_screenshot('callBarring-init')
-        cs_page.wait_until_call_barring_info_received()
+        cs_page.wait_until_call_barring_info_received()  # this sometimes happens in less than a second,
+                                                         # making it difficult to obtain init view
```

Comment:
yes, by the time the screenshot is taken, it already exited the wait state.  and the duration fluctuates too, sometimes in less than 1/2 seconds, sometimes takes 10 seconds.